build.assets/Makefile

```diff
@@ -328,3 +277,28 @@ release-arm64-fips: buildbox-arm-fips
 	@if [ -z ${VERSION} ]; then echo "VERSION is not set"; exit 1; fi
 	docker run $(DOCKERFLAGS) $(BCCFLAGS) -i $(NOROOT) $(BUILDBOX_ARM_FIPS) \
 		/usr/bin/make -C e release -e ADDFLAGS="$(ADDFLAGS)" OS=$(OS) ARCH=$(ARCH) RUNTIME=$(RUNTIME) FIPS=yes VERSION=$(VERSION) GITTAG=v$(VERSION)
+
+#
+# Run docs tester to detect problems.
+#
+.PHONY: docsbox
+docsbox:
+	docker pull $(DOCSBOX)
+
+.PHONY: test-docs
+test-docs: MAKEFILE_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+test-docs: MAKEFILE_ROOT_DIR := $(shell readlink -f $(MAKEFILE_DIR)/..)
+test-docs: DOCS_VERSION := $(shell grep -E ^VERSION $(MAKEFILE_ROOT_DIR)/Makefile | cut -d= -f2 | cut -d. -f1-2)
+test-docs: DOCS_TEMP := $(shell mktemp -d --tmpdir=$(MAKEFILE_ROOT_DIR))
+test-docs: DOCS_TEMP_DIRNAME := $(shell basename $(DOCS_TEMP))
+test-docs: docsbox
+	@chown -R $(UID):$(GID) $(DOCS_TEMP)
+	cd $(DOCS_TEMP) && \
+		git clone https://github.com/gravitational/next . && \
+		rm -rf content/$(DOCS_VERSION) && \
+		ln -s $(MAKEFILE_ROOT_DIR) content/$(DOCS_VERSION)
+	bash -c " \
+		trap '{ rm -rf $(DOCS_TEMP); }' EXIT; \
```

Comment:
this should happen as the first step in this make target, as close to `mktemp` as possible